searx/templates/simple/base.html

```diff
@@ -77,6 +77,11 @@
         {% if get_setting('general.contact_url') %}
         | <a href="{{ get_setting('general.contact_url') }}">{{ _('Contact instance maintainer') }}</a>
         {% endif %}
+        {% for item in get_setting('brand.custom') %}
+          {% for key, value in item.items() %}
+          | <a href="{{ value }}">{{ key }}</a>
+          {% endfor %}
+        {% endfor %}
```

Comment:
I don't really like that nested loop here, but we first need to iterate over a `list`, and then over the `dict` items inside.

Reply:
Haven't tested but I would expect that YAML construct is what you are looking for:

```yaml
custom:
 About instance: "https://searxng.org"
 Instance admin: "https://searxng.org"
```

Reply:
Works, thank you!